src/pages/contact.tsx

```diff
@@ -21,11 +22,13 @@ const ContactPage: React.FC = () => {
   return (
     <>
       <Aurora type={AuroraType.Pink} />
-      <Layout transparentHeader={true}>
+
+      <Layout transparentHeader={true} hero={<Leaflet />}>
```

Comment:
ne option `hero` allows us to pass in any component to be placed between the fixed header and the content.